src/07_slices.py

```diff
@@ -12,26 +12,33 @@
 a = [2, 4, 1, 7, 9, 6]
 
 # Output the second element: 4:
-print()
+x = slice(1,2)
+print(a[x])
```

Comment:
Hey, @travislittle84! Nice work! You could have also done this like `print(a[1])`! 👍